packages/wallets/localize-loader.js

```diff
@@ -31,10 +38,16 @@ module.exports = async function (source) {
                                 key
                             )} and ${value}.`
                         );
-                    path.node.arguments[0] = {
-                        type: 'StringLiteral',
-                        value: key,
-                    };
+
```

Comment:
on the generateKey function - there is that .substring(0, 8) in it, can we remove it? or is it needed? feels like its not needed, yet increases chances of collisions a lot (actually we had almost the same problem in previous place, but with assets hashing, and became unbearable suprisingly quickly).
Also, if there is high risk of collisions, then build should explicitly fail in case of collision - so thats yet another risk / feature to maintain to ensure system integrity.

Reply:
The reason I substring-ed it is because I didn't want the hashed keys to have more than needed characters, it might increase the bundle size because short strings like `Back` can be hashed into `76900f1bfd16c8d4dd3d25e6f46638d7165aee23883ccea6bfe071c514421769`, and this is included in the bundle in build time

Reply:
I think in this case you may be right, it does increase the chance of collision:
https://stackoverflow.com/questions/28671639/reducing-size-of-hash
Although the chances of collision is still pretty low, we are using CRC32 for generating the keys and its a more suitable key hasher and only generates keys of length 8, similar to this method as well
I'll apply the CRC32 method for this case, thanks for the catch 🙏